docs/configuration.md

```diff
@@ -211,4 +211,50 @@ telemetries: [
         }
     ],
 ]
-```
\ No newline at end of file
+```
+
+## Time to Interactive (TTI)
+
+#### What is Time to interactive (TTI)?
+
+Time to interactive (TTI) is a measure of how long a page takes till it is interactive. AWS RUM Web Client provides a plugin you can use to record this measurement. 
+
+#### How is TTI measured by the AWS RUM web client? 
+
+The web client TTI plugin implements the [*Boomerang TTI algorithm*](https://akamai.github.io/boomerang/oss/BOOMR.plugins.Continuity.html#toc11__anchor), with some modifications. 
```

Comment:
> with some modifications

suggestion: explicitly state that we have omitted page busy metrics for performance concerns